components/gitlab/new-project.js

```diff
@@ -0,0 +1,89 @@
+const gitlab = require("https://github.com/PipedreamHQ/pipedream/components/gitlab/gitlab.app.js");
+
+module.exports = {
+  name: "New Project",
+  description: "Triggers when a new project (i.e. repository) is created",
+  version: "0.0.1",
+  dedupe: "greatest",
+  props: {
+    gitlab,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 15 * 60, // 15 minutes
+      },
+    },
+    http: "$.interface.http",
+    db: "$.service.db",
+  },
+  hooks: {
+    async activate() {
+      const opts = {
+        pageSize: 1,
+      };
+      const projects = await this.collectProjects(opts);
```

Comment:
Am I misreading? We only need the latest project here but it looks like you'd collect all the user's projects but with 1 per_page? Would be nice to just make call for the first page.

Reply:
I think you're right, good catch!! I'll change this to use what `getCommits` already does (limit the amount of queried items).

Reply:
Indeed you were right. Just pushed a fix.